repository/RediStick-Connection-Pool.package/RsRedisConnectionPool.class/instance/createStickAt..st

```diff
@@ -5,5 +5,6 @@ createStickAt: key
 	| stick |
 	stick := RsRediStick targetUrl: self primaryUrl.
 	stick beSticky.
+	stick username: self userName password: self password.
```

Comment:
`username:` should be `userName:` for consistency.

Reply:
fix 35150ae